gradle-plugins/src/main/java/com/linkedin/pegasus/gradle/tasks/ValidateSchemaAnnotationTask.java

```diff
@@ -198,10 +198,16 @@ private void checkHandlerAnnotation(String name, List<String> foundClasses) thro
             return;
           }
         }
-      } catch (ClassNotFoundException e)
+      }
+      catch (ClassNotFoundException e)
       {
         throw new IOException("Failed to load class while scanning classes", e);
       }
+      catch (Exception | Error e)
+      {
+        getProject().getLogger()
+                    .info("Unexpected exceptions or errors found during scanning class, ok to skip", e);
```

Comment:
Why is it okay to skip?

Reply:
Sorry I found I made a mistake regarding the content to publish. Let me create a separate view for above change.

I want this review to only focus on PathSpec key type change